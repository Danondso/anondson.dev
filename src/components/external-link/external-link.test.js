import React from 'react';
import renderer from 'react-test-renderer';
import ExternalLink from './external-link';

describe('External Link Snapshot test', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <ExternalLink url="http://localhost:8080"> LocalHost </ExternalLink>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
