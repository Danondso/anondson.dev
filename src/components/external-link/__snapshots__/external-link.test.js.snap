// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`External Link Snapshot test renders correctly 1`] = `
<a
  className="link"
  href="http://localhost:8080"
  rel="noopener noreferrer"
  target="_blank"
>
   LocalHost 
</a>
`;
